devel/doctest/Portfile

```diff
@@ -0,0 +1,18 @@
+PortSystem 1.0
+PortGroup cmake 1.0
+PortGroup github 1.0
+github.setup        onqtam doctest 2.4.6 v
+github.tarball_from archive
+
+revision 0
+categories devel
+maintainers linux.com:nickblack
+description Fast header-only C++ unit testing
+long_description \
+ doctest is a light and feature-rich C++98 / C++11 single-header testing \
+ framework for unit tests and TDD.
+homepage https://github.com/onqtam/doctest
```

Comment:
homepage is not necessary here as it is set automatically by the `github` PG

Reply:
yanked, thanks